wallaby.js

```diff
@@ -17,12 +17,10 @@ module.exports = wallaby => ({
   files: [
     'jest.config.js',
     'packages/*/src/**/*.{ts,tsx,snap}',
+    '!packages/{dom,demos}/**/*',
     '!packages/*/src/**/*.test.{ts,tsx}'
   ],
-  tests: [
-    'packages/*/src/**/*.test.{ts,tsx}',
-    '!packages/demos/src/**/*.test.{ts,tsx}'
-  ],
+  tests: ['packages/*/src/**/*.test.{ts,tsx}', '!packages/{dom,demos}/**/*'],
```

Comment:
`!packages/{dom,demos}/**/*.{ts,tsx,snap}`

`!packages/{dom,demos}/**/*.{ts,tsx}`

Reply:
Why?

Reply:
Our intend is to exclude the `dom` and `demo` packages from wallaby regardless of file extensions.